catalog-rest-service/src/main/resources/json/schema/api/tests/createTableTest.json

```diff
@@ -0,0 +1,41 @@
+{
+  "$id": "https://open-metadata.org/schema/api/tests/tableTest.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "CreateTableTestRequest",
+  "description": "TableTest is a test definition to capture data quality tests against tables and columns.",
+  "type": "object",
+  "properties": {
+    "description": {
+      "description": "Description of the testcase.",
+      "type": "string"
+    },
+    "tableName": {
+      "description": "Table Name for which this test applies.",
+      "type": "string"
+    },
+    "tableTestCase": {
```

Comment:
I'd either rename this to `testCase` (as we currently have with `ColumnTest`) or rename Column's `testCase` to `columnTestCase` to follow the same naming convention